### PR TITLE
Ethereum studio v2

### PR DESCRIPTION
**Pull request by samajammin**
## Description
Building off @rrecuero's great work in https://github.com/ethereum/ethereum-org-website/pull/308 (thanks Ramon!), add a "Get Building" page for smoother developer onboarding.

## Related Issue
There doesn't appear to be one... 😑 

## Screenshots (if appropriate):
See preview deployment, page /start-building

start-building

## Checklist:

- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
Whoops, pulling in comments from old PR…

https://github.com/ethereum/ethereum-org-website/pull/308#discussion_r335910410

Just really the fungible term, it's OK, but I do wonder how niche it is to our ecosystem.

File: docs/.vuepress/components/StartBuildingPage.vue [hunk 1]
```diff
@@ -0,0 +1,255 @@
+<template>
+  <div class="start-building">
+    <div class="center">
+      <div class="headline-text-container">
+        <h1 class="headline-title">Get started building on Ethereum</h1>
+        <div class="headline-subtitle">Learn the basics of Ethereum with Ethereum Studio, our web-based IDE where you can create and test smart contracts and their front end.</div>
```
Comment (ChrisChinchilla) [inline docs/.vuepress/components/StartBuildingPage.vue]:
```suggestion
        <div class="headline-subtitle">Learn the basics of Ethereum with Ethereum Studio, our web-based IDE where you can create and test smart contracts, and build a front end for them.</div>
```

Something like this, it felt like we were saying "test the front end"

**Comment by samajammin** [comment]
This is good to go, just waiting on finishing touches with [Ethereum Studio](https://github.com/SuperblocksHQ/ethereum-studio).

**Comment by jjmstark** [comment]
@samajammin just noticed one change needed:

the "Artwork by...." sentence is still in the footer here, but it should be removed. On site right now, that only displays on the home page, but not on sub-pages.

**Comment by samajammin** [comment]
Nice catch @jjmstark - updated to only show that on the home page.

**Pull request merged**